src/lucky/redirectable.cr

```diff
@@ -1,19 +1,67 @@
+# Redirect the request
+#
+# There are multiple ways to redirect inside of an action. The most common ways are to redirect to a `Lucky::Action` class, or a URL/path `String`. Both use the `redirect` method:
+#
+# ```crystal
+# redirect to: Users::Index
+# redirect to: Users::Show.with(user.id)
+# redirect to: "https://luckyframework.org/"
+# redirect to: "/users"
+# ```
+#
+# By default, the method will set the status code to `302` A.K.A. "Found". If you want to customize the status code, you can pass it directly:
+#
+# ```crystal
+# redirect to: Users::Index, status: 301
+#
+# # or use the built in enum value
+# redirect to: Users::Index, status: Status::MovedPermanently
+# ```
+#
+# You can find a list of all of the possible statuses [here](https://github.com/luckyframework/lucky/blob/master/src/lucky/action.cr).
```

Comment:
Not the best link, but it seemed better than linking to a soon-to-be out of date commit or a line number that might change.

Reply:
👍 This should work